app/templates/base.html

```diff
@@ -79,7 +83,18 @@
     
 </div>
 
+<<<<<<< HEAD
+<!---- footer--> 
+<div class="footer">
+    <link><a href="privacy_policy.html" > Privacy Policy</a></link>
+    <link><a href="about_us.html" > About Us</a></link>
+    <link><a href="blog.html" > Blog</a></link>  <br>
+    <link><a href="terms.html" > Terms</a></link>
+    <link><a href="contact_us" > Contact Us</a></link>
+</div>
+=======
```

Comment:
same again, naming doesnt match ? base file its contact_us.html and in the templates folder its saved as contact.html